cmd/contour/serve.go

```diff
@@ -24,6 +24,8 @@ import (
 	"syscall"
 	"time"
 
+	xds "github.com/envoyproxy/go-control-plane/pkg/cache/types"
```

Comment:
If you add an alias for `ResourceType` to the existing `xds` package, then you can avoid the import conflict here and in other places. I'd recommend aliasing it as `ResourceType`, which indicates its usage better.

Reply:
Something like this? 

```
type ResourceType = xds.ResponseType
```

How do I get to the enumeration types? 

This is the struct:

```
// ResponseType enumeration of supported response types
type ResponseType int

const (
	Endpoint ResponseType = iota
	Cluster
	Route
	Listener
	Secret
	Runtime
	UnknownType // token to count the total number of supported types
)
```